src/consensus/consensus.h

```diff
@@ -6,11 +6,105 @@
 #ifndef BITCOIN_CONSENSUS_CONSENSUS_H
 #define BITCOIN_CONSENSUS_CONSENSUS_H
 
+#include "amount.h"
+#include "consensus/params.h"
+
+#include <stdint.h>
+
+class CBlock;
+class CBlockHeader;
+class CBlockIndex;
+class CCoinsViewCache;
+class CTransaction;
+class CValidationState;
+
+/**
+ * Consensus validations:
+ * Check_ means checking everything possible with the data provided.
+ * Verify_ means all data provided was enough for this level and its "consensus-verified".
+ */
+namespace Consensus {
+
+/** Transaction validation functions */
+
+/**
+ * Context-independent CTransaction validity checks
+ */
+bool CheckTx(const CTransaction& tx, CValidationState& state, const Params& consensusParams);
+/**
```

Comment:
@jtimon it helps to distinguish blocks apart from each other,  and is the standard adopted by most of the code base as far as I can tell.